app/forms/onboarding/api_form.rb

```diff
@@ -29,6 +29,14 @@ def example_backend
     ECHO_API_BACKEND
   end
 
+  def save
```

Comment:
I do not think we need this

The onboarding does not create a service, it just updates the ones created in ProviderSignup#signup_provider

Reply:
Makes total sense